webapp/src/main/java/com/epam/esm/config/WebConfig.java

```diff
@@ -17,7 +16,7 @@
 @Configuration
 @ComponentScan(basePackages = {"com.epam.esm"})
 @PropertySource("classpath:webapp.properties")
```

Comment:
Read about @configurationProperties annotation in SpringBoot

Reply:
Created class with properties